src/app/test/db/mongo/models/models_test.py

```diff
@@ -6,7 +6,7 @@
 from brewtils.errors import ModelValidationError, RequestStatusTransitionError
 from brewtils.schemas import RequestTemplateSchema
 from mock import Mock
-from mongoengine import ValidationError, connect
+from mongoengine import NotUniqueError, ValidationError, connect
```

Comment:
No idea how this got omitted despite what appeared to be correct rebasing. At any rate, ignore. It's what a previously approved PR added.